CesiumAsync/src/QueuedScheduler.cpp

```diff
@@ -3,17 +3,59 @@
 #include <condition_variable>
 #include <mutex>
 
-// Hackily use Async++'s internal fifo_queue. We could copy it instead - it's
-// not much code - but why create the duplication? However, we are assuming that
-// Async++'s source (not just headers) are available while building
-// cesium-native. If that's a problem in some context, we'll need to do that
-// duplication of fifo_queue after all.
-#include <async++/../../src/fifo_queue.h>
+namespace {
+
+// This class is copied from async++'s fifo_queue.h:
```

Comment:
Maybe add a note about Async++'s source not being available when building `cesium-native`? (As the old comment suggests)